content/docs/concepts/security/index.md

```diff
@@ -36,7 +36,7 @@ bare-metal machines.
     caption="Istio Security Architecture"
     >}}
 
-As illustrated in the diagram, Istio leverages secret volume mount to deliver
+As illustrated in the diagram, Istio leverages secret volume mounts to deliver
```

Comment:
please don't use the word leverage.  Try "Istio Uses"

Reply:
Done